boards/x86/qemu_x86/qemu_x86.dts

```diff
@@ -39,9 +40,12 @@
 	};
 
 	soc {
-		eth0: eth@febc0000 {
+		eth0: eth@1800 {
 			compatible = "intel,e1000";
-			reg = <0xfebc0000 0x100>;
+
```

Comment:
If we are going to make this change, I assume there is PCIe bus controller on the SoC.  We really should add the PCIe controller and move the e1000 node under that controller.

Reply:
I think that is outside of scope of this PR.